spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/web/servlet/error/DefaultErrorAttributes.java

```diff
@@ -94,7 +102,7 @@ private void storeErrorAttributes(HttpServletRequest request, Exception ex) {
 	private void addStatus(Map<String, Object> errorAttributes,
 			RequestAttributes requestAttributes) {
 		Integer status = getAttribute(requestAttributes,
-				"javax.servlet.error.status_code");
+				RequestDispatcher.ERROR_STATUS_CODE);
```

Comment:
Yes, that's what I was referring to in the opening comment:

> I've also changed DefaultErrorAttributes to use RequestDispatcher constants instead of String literals for standard attribute names.

Should I open a separate PR to address this?